Templates/WeakAurasTemplates.toc

```diff
@@ -0,0 +1,25 @@
+## Interface: 70000
+## Title: WeakAuras Templates
+## Author: Infus
+## Version: @project-version@
+## Notes: Templates for WeakAuras
+## X-Category: Interface Enhancements
+## DefaultState: Enabled
+## LoadOnDemand: 1
+## Dependencies: WeakAuras, WeakAurasOptions
+
+Localization-enUS.lua
```

Comment:
locale files are missing atm, just copy paste them over from Tutorials and replace `Tutorials` with `Templates` in them, I will hook it up with the translation system then